### DISCUSSION TITLE
Welcome to ActivityWatch Discussions!

### DISCUSSION THREAD

**Discussion by ErikBjare** [ActivityWatch/activitywatch#527] [General]
## 👋 Welcome!
  We're just trying out this GitHub Discussions thing. 

  We still prefer you use [our forum](https://forum.activitywatch.net/) for feature requests and general support, but we think this place might serve for more technical discussions that aren't well suited for issues and doesn't have a natural home on the forum.

  If you aren't sure where to post, check out the [new issue chooser](https://github.com/ActivityWatch/activitywatch/issues/new/choose) which will point you in the right direction.

  We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gogolian** [comment]
Hello Everyone :)

**Comment by diesacademicus** [comment]
Just downloaded ActivityWatch, after seeing someone mention it in a comment on Reddit (comment was a bit older).

Was and am currently using procrastitracker, so I actually have a pretty good comparison. Will give feedback after some usage time. Thanks folks.
